src/main/java/sirius/biz/jobs/batch/file/XMLImportJobFactory.java

```diff
@@ -42,12 +49,14 @@ public String getIcon() {
     }
 
     /**
-     * Returns the path to the XSD file if the XML file should be validated, null otherwise.
+     * Returns a map of paths to XSD files if the XML file should be validated, an empty map otherwise.
+     * <p>
+     * This allows to provide more than one XSD for validation when multiple variants of a data format are supported.
      *
-     * @return the path to the XSD file if the XML file should be validated, null otherwise
+     * @return a map of paths to XSD files if the XML file should be validated, an empty map otherwise
      */
-    @Nullable
-    protected String getXsdResourcePath() {
-        return null;
+    @Nonnull
+    protected Map<String, String> getXsdResourcePaths() {
```

Comment:
create a map externally and offer it here as BiConsumer<String,String> here? this also permits to enforce a LinkedHashMap which guarantees order